apps/site/pages/en/about/index.mdx

```diff
@@ -77,3 +77,45 @@ by using our [`child_process.fork()`](https://nodejs.org/api/child_process.html)
 communicate with. Built upon that same interface is the [`cluster`](https://nodejs.org/api/cluster.html) module,
 which allows you to share sockets between processes to enable load balancing
 over your cores.
+
+## Official Node.js Resources
+
+To ensure authenticity and security when working with Node.js, always use official sources. Avoid trusting emails,
+binaries, or downloads from unofficial sources.
+
+### Official Node.js Domains
+
+For downloading Node.js binaries and accessing official documentation, use only these domains:
+
+- [nodejs.org](https://nodejs.org)
+- [nodejs.dev](https://nodejs.dev) _(Redirects to https://nodejs.org)_
+- [iojs.org](https://iojs.org) _(Redirects to https://nodejs.org)_
+
+### Official npm Packages
+
+The Node.js team maintains the following official npm package scopes:
+
+- [`@node-core`](https://npmjs.com/~node-core)
+- [`@pkgjs`](https://npmjs.com/~pkgjs)
+
+Additionally, the Node.js team maintains packages published by the [`nodejs-foundation`](https://npmjs.com/~nodejs-foundation) npm account,
+though other Node.js-related packages (like Ada) may also be maintained by contributors closely tied to the project.
```

Comment:
I would recommend mentioning `readable-stream` and `undici` as they have _a lot_ of usage and it would be worthwhile listing them here specifically.

Reply:
Will do! (Also, ada's not an npm package, so it doesn't belong there anyway)

Reply:
readable-stream is on the foundation account (as a prior publisher was that account), but I've added Undici)